src/css/components/_keyboard.scss

```diff
@@ -4,7 +4,29 @@
     display: flex;
     justify-content: center;
     flex-direction: column;
-    padding: 20px;    
+    padding: 20px;
+    padding-top: 0px;
+}
+
+.keyboard .vscrollmenu {
```

Comment:
Might have to limit the height of the vscrollitem primary image as well
![Screenshot from 2022-02-02 14-47-46](https://user-images.githubusercontent.com/10703941/152357365-f4ca3965-c1b7-40f6-9187-e98e7b3c0bbc.png)